test/e2e/tests/hardware-wallets/ledger-erc20.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { Suite } from 'mocha';
+import { Driver } from '../../webdriver/driver';
+import TestDappPage from '../../page-objects/pages/test-dapp';
+import FixtureBuilder from '../../fixture-builder';
+import { WINDOW_TITLES, withFixtures } from '../../helpers';
+import { KNOWN_PUBLIC_KEY_ADDRESSES } from '../../../stub/keyring-bridge';
+import { loginWithoutBalanceValidation } from '../../page-objects/flows/login.flow';
+import CreateContractModal from '../../page-objects/pages/dialog/create-contract';
+
+describe('Ledger Hardware', function (this: Suite) {
+  it('can create an ERC20 token', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder()
+          .withLedgerAccount()
+          .withPermissionControllerConnectedToTestDapp({
+            account: KNOWN_PUBLIC_KEY_ADDRESSES[0].address,
+          })
+          .build(),
+        title: this.test?.fullTitle(),
+        dapp: true,
+      },
+      async ({ driver, localNodes }) => {
+        (await localNodes?.[0]?.setAccountBalance(
+          KNOWN_PUBLIC_KEY_ADDRESSES[0].address,
+          '0x100000000000000000000',
+        )) ?? console.error('localNodes is undefined or empty');
+        await loginWithoutBalanceValidation(driver);
+        const testDappPage = new TestDappPage(driver);
```

Comment:
to prevent flakiness, could we make sure we have the expected balance once we land in the hompage?
Something like:

```suggestion
        const homePage = new HomePage(driver);
        await homePage.check_expectedBalanceIsDisplayed('1208925.8196');
        const testDappPage = new TestDappPage(driver);
```